src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -662,5 +662,7 @@ public override IOperation VisitUsingDeclaration(IUsingDeclarationOperation oper
         {
             return new UsingDeclarationOperation(Visit(operation.DeclarationGroup), operation.IsAsynchronous, ((Operation)operation).OwningSemanticModel, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
+
+        // TODO2
```

Comment:
If work still needs to be done here, could you please create a follow-up issue and reference it instead of including a TODO in source?

Reply:
Oops. I'll remove the comment and file an issue.
I think `OperationCloner` is used by CFG, we'll probably need to do some work there for records.

Reply:
Added a note to test plan